ocs_ci/ocs/ui/validation_ui.py

```diff
@@ -0,0 +1,155 @@
+        self.verify_page_contain_strings(
+            strings_on_page=strings_details_tab, page_name="details_tab"
+        )
+
+        logger.info("Verify Subscription tab on OCS operator")
+        self.do_click(self.validation_loc["osc_subscription_tab"])
+        strings_subscription_tab = [
+            "Healthy",
+            "openshift-storage",
+            "Channel",
+            "Approval",
+        ]
+        self.verify_page_contain_strings(
+            strings_on_page=strings_subscription_tab, page_name="subscription_tab"
+        )
+
+        logger.info("Verify All instances tab on OCS operator")
+        self.do_click(self.validation_loc["osc_all_instances_tab"])
+        strings_all_instances_tab = ["Phase", "Ready", "Status"]
+        self.verify_page_contain_strings(
+            strings_on_page=strings_all_instances_tab, page_name="all_instances_tab"
+        )
+
+        logger.info("Verify Storage Cluster tab on OCS operator")
+        self.do_click(self.validation_loc["osc_storage_cluster_tab"])
+        strings_storage_cluster_tab = ["Phase", "Ready", "Status"]
+        self.verify_page_contain_strings(
+            strings_on_page=strings_storage_cluster_tab, page_name="storage_cluster_tab"
+        )
+
+        logger.info("Verify Backing Store tab on OCS operator")
+        self.do_click(self.validation_loc["osc_backing_store_tab"])
+        strings_backing_store_tab = ["Phase", "Ready", "Status"]
+        self.verify_page_contain_strings(
+            strings_on_page=strings_backing_store_tab, page_name="backing_store_tab"
+        )
+
+        logger.info("Verify Bucket Class tab on OCS operator")
+        self.do_click(self.validation_loc["osc_bucket_class_tab"])
+        strings_bucket_class_tab = ["Phase", "Ready", "Status"]
+        self.verify_page_contain_strings(
+            strings_on_page=strings_bucket_class_tab, page_name="bucket_class_tab"
+        )
+
+    def verify_page_contain_strings(self, strings_on_page, page_name):
+        """
+        Verify Page Contain Strings
+
+        Args:
+            strings_on_page (list): list of strings on page
+            page_name (str): the name of the page
+
+        """
+        logger.info(f"verify {strings_on_page} exist on {page_name}")
+        for string in strings_on_page:
+            sample = TimeoutSampler(
+                timeout=3,
+                sleep=1,
+                func=self.check_element_text,
+                expected_text=string,
+            )
+            if not sample.wait_for_func_status(result=True):
+                self.err_list.append(f"{string} string not found on {page_name}")
+
+    def verification_ui(self):
+        """
+        Verification UI
+
+        """
+        self.verify_object_service_page()
+        self.verify_persistent_storage_page()
+        self.verify_ocs_operator_tabs()
+        self.take_screenshot()
+        for err in self.err_list:
+            logger.error(err)
```

Comment:
Maybe add that after you check the length of self.err_list. If = 0 ..... else do this lines

Reply:
If `self.err_list` list is empty, these lines does not execute